test/conformance.go

```diff
@@ -62,6 +63,8 @@ const (
 // Setup creates client to run Knative Service requests
 func Setup(t pkgTest.TLegacy) *Clients {
 	t.Helper()
+	t.Cleanup(logstream.Start(t))
```

Comment:
Should we make this 2 lines? I tripped over this :sweat_smile:

Reply:
I bet you looooooved this 😉 
```
defer logstream.Start(t)()
```

Yeah, 2 lines here is way cheaper than the 3 or so it was (and would have been) everywhere else.